app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,87 @@
+class OrdersController < ApplicationController
+  before_action :require_login, only: [:show_seller_orders]
+
+  def show
+    @orders = Order.find(current_order.id).orderitems
+  end
+
+  def show_seller_orders
+  	@user = User.find(current_user.id)
+    @user_orders_hash = Orderitem.where(user: current_user).group_by(&:order_id)
+    @revenue = @user.revenue
+    @completed_revenue = @user.revenue_by_status("Completed")
+    @pending_revenue = @user.revenue_by_status("Pending")
+    @completed_count = @user.order_by_status("Completed")
+    @pending_count = @user.order_by_status("Pending")
+  end
+
+  def order_deets
+    @user_orders_hash = Orderitem.where(user: current_user).group_by(&:order_id)
+    @order = Order.find(params[:order_id])
+  end
+
+
+  def edit
+    @order  = Order.find(params[:id])
+    @orderitems = Order.find(current_order.id).orderitems
+  end
+
+  def update
+    @order = current_order
```

Comment:
I'm noticing that in some places you do ```@order = current_order``` and in some you do ```@order = Order.find(params[:order_id])```. Is there a reason you're switching between them?